Formula/google-protobuf.rb

```diff
@@ -59,6 +65,7 @@ def install
     ENV.universal_binary if build.universal?
     ENV.cxx11 if build.cxx11?
 
+    (buildpath/"gmock").install resource("gmock")
```

Comment:
Can you point me to some documentation on how this works? I am not too familiar with this particular syntax. How does `.install` seem to know to unzip the zip file?